Source/WebKit/NetworkProcess/NetworkProcess.h

```diff
@@ -72,7 +72,7 @@ typedef struct OpaqueCFHTTPCookieStorage*  CFHTTPCookieStorageRef;
 #endif
 
 #if USE(EXTENSIONKIT)
-OBJC_CLASS WKGrant;
+OBJC_CLASS NSObject;
```

Comment:
Is it possible to define a protocol matching the pieces of WKGrant we need, so that we retain more type safety than this generic NSObject approach?

At least then we could define things in terms of `NSObject <WKGrantProtocol>` or something.

Reply:
That is a good point. The generic NSObject approach is added to avoid the dependency cycle, so I believe this is required to fix the issue.

Thanks for reviewing!